### ISSUE TITLE
Able to add Orders through our dashboard.. in case the customer just calls and does not place through his login 

### ISSUE THREAD

**Issue by cricketabhi** [harish-myaccount/localmarket#15]
Initially no one is using the customer side page.. it has to happen over call

**Comment by harishreddy-m** [comment]
Admin can login with customer credentials and order the item.

**Comment by cricketabhi** [comment]
no.. not a good idea. passwords are stored securely and even an admin like
us should not be able to access them

_-----------------------------_
_Abhishek Susarla_

**Comment by harishreddy-m** [comment]
If u add items into customer it also violates integrity of customer data...
Imagine a customer complaining "I never ordered this.I was billed wrong"

**Comment by cricketabhi** [comment]
Ala aythe vaadu dabbulu ivvadu ra....  So.. The risk is on us... Dont
worry.. Ee feature kavali

**Comment by harishreddy-m** [comment]
For that we have to fetch items,display... Redo of whole display page of
customer
Anyways this feature is for people who are not using site...then why not
login as customer and add order...

**Comment by cricketabhi** [comment]
Display side of customer enduku change cheyyali...
1. Customer is able to edit / modify orders
2. MOA is able to edit / modify orders..

anyways ippudu manam customer ki aa feature ichham kadha.. mana side
matrame maarchali...

i think an order should be accessiblet from both endths. and its a must
have feature

**Comment by harishreddy-m** [comment]
I didn't say..change customer end...

whatever done on customer end..has to repeated on admin end too

**Comment by cricketabhi** [comment]
lets do it. ee weekend ki without CSS the basic functionality has to be
ready. we are getting a lot of queries and hiring is going on too..
elagaina ee features cheyyu ee weekend kalla

_-----------------------------_
_Abhishek Susarla_
